src/ec/packages/ec-component-language-list/language-list.story.js

```diff
@@ -11,7 +11,7 @@ import dataSplash from './demo/data--splash';
 import dataOverlay from './demo/data--overlay';
 
 import languageListSplash from './ecl-language-list-splash.html.twig';
```

Comment:
if we use now one template maybe this _splash_ file needs to be renamed to `ecl-language-list-main.html.twig`?
`languageListSplash` can also be renamed to `languageList`?

Reply:
you need also to remove `ecl-language-list-overlay.html.twig` file if it's no used anymore.

Reply:
I left it like this for maybe older versions, who will still look for a twig template. But sure I will remove it.